docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<p>Overall, this notebook needs a bit more commentary.</p><p> </p><p>We should include a sentence or two here explaining what exactly the notebook demonstrates.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Added some comments